monitoring/serializers.py

```diff
@@ -15,11 +15,15 @@ class Meta:
         id_field = False
         fields = (
             'id',
-            'tb_ciclo_monitoramento_id',
             'no_imagem',
             'dt_imagem',
             'no_estagio',
-            'dt_cadastro',
+            'no_ciclo',
```

Comment:
Do you guys, really need all fields on properties?

The behave expected when you click on geometry is to call an api request, right? 
So... I think that you dont need all fields here. Less is more!

Reply:
OK